cmake/projects/CURL/hunter.cmake

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2015, David Hirvonen
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+# Load used modules
+
+
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+
+hunter_add_package(OpenSSL)
+hunter_add_package(ZLIB)
+hunter_add_package(LibSSH2)
+# List of versions here...
+
+hunter_add_version(
+    PACKAGE_NAME
+    CURL
+    VERSION
+    "7.49.1-DEV"
+    URL
+    "https://github.com/hunter-packages/curl/archive/hunter-7.49.1-v0.zip"
+    SHA1
+    fd72a153ca60fe85b57a2ac4eadedcfb38c3066d
+)
+
+
+if(BUILD_SHARED_LIBS)
```

Comment:
Should not be here. `BUILD_SHARED_LIBS` is for controlling **current** project, not 3rd party. CURL should respect default CMake feature: `BUILD_SHARED_LIBS`. See: https://github.com/ruslo/hunter/wiki/usr.adding.new.package#default-behaviour